src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const images = document.getElementById('thumbs');
+const bigImage = document.getElementById('largeImg');
+
+images.addEventListener('click', (someEvent) => {
+  const link = someEvent.target.closest('.list-item__link');
+
+  if (!link || !images.contains(link)) {
+    return;
+  }
+
+  someEvent.preventDefault();
```

Comment:
It's more understandable if you write .preventDefault() right At the beginning of your code inside the listener
(on line 7)